lib/ansible/modules/cloud/amazon/lambda.py

```diff
@@ -96,6 +96,12 @@
     description:
       - The parent object that contains the target Amazon Resource Name (ARN) of an Amazon SQS queue or Amazon SNS topic.
     version_added: "2.3"
+  tracing_mode:
```

Comment:
Do you mind adding this to the return documentation and updating the aws_lambda test target to use it?

Reply:
Done now